dpe/src/lib.rs

```diff
@@ -31,7 +31,7 @@ use zerocopy::{FromBytes, Immutable, IntoBytes, KnownLayout, TryFromBytes};
 pub use crypto::{ecdsa::EcdsaAlgorithm, ExportedCdiHandle, MAX_EXPORTED_CDI_SIZE};
 
 // Max cert size returned by CertifyKey
-const MAX_CERT_SIZE: usize = 6144;
+const MAX_CERT_SIZE: usize = 7872;
```

Comment:
I think we discussed this a little bit before, but we may want to consider a chunking mechanism for retrieving certs in 2.x. Ok to leave out of this PR, but could you open an issue?